src/facade/arg_parser.h

```diff
@@ -36,19 +36,19 @@ struct ArgumentParser {
     }
 
     template <typename T> operator T() {
```

Comment:
I'd prefer this to be a named function (`ParseInt<T>` or something) and not an implicit cast.

Reply:
Why? Isn't it always deduced correctly? Otherwise you need to do `Next().ParseInt<uint64_t>()` specifying the type

Reply:
Why? Isn't it always deduced correctly? Otherwise you need to do `Next().ParseInt<uint64_t>()` specifying the type

Reply:
I just don't like implicit operations that can fail :)